### PR TITLE
[CARBONDATA-353]Update doc for dateformat option

### PR DESCRIPTION
**Pull request by lion-x**
# Why raise this PR?

Update doc for dateformat option
# CI Passed

http://136.243.101.176:8080/job/ApacheCarbonManualPRBuilder/535/

### CODE REVIEW

**Comment by lion-x** [comment]
@jackylk

File: docs/DML-Operations-on-Carbon.md [hunk 1]
```diff
@@ -91,12 +91,17 @@ Following are the options that can be used in load data:
     ```ruby
     OPTIONS('ALL_DICTIONARY_PATH'='/opt/alldictionary/data.dictionary')
     ```
-- **COLUMNDICT:** dictionary file path for single column.
+- **COLUMNDICT:** Dictionary file path for each column.
```
Comment (jackylk) [inline docs/DML-Operations-on-Carbon.md]: change `each` to `specified`

File: docs/DML-Operations-on-Carbon.md [hunk 2]
```diff
@@ -91,12 +91,17 @@ Following are the options that can be used in load data:
...
 
     ```ruby
     OPTIONS('COLUMNDICT'='column1:dictionaryFilePath1, column2:dictionaryFilePath2')
     ```
     Note: ALL_DICTIONARY_PATH and COLUMNDICT can't be used together.
+- **DATEFORMAT:** Date format for each column.
```
Comment (jackylk) [inline docs/DML-Operations-on-Carbon.md]: change `each` to `specified`

File: docs/DML-Operations-on-Carbon.md [hunk 3]
```diff
@@ -91,12 +91,17 @@ Following are the options that can be used in load data:
...
+
+    ```ruby
+    OPTIONS('DATEFORMAT'='column1:dateFormat1, column2:dateFormat2')
```
Comment (jackylk) [inline docs/DML-Operations-on-Carbon.md]: give an example of the data format

File: docs/DML-Operations-on-Carbon.md (hunk 1 above)
Comment (lion-x) [inline docs/DML-Operations-on-Carbon.md]: updated

File: docs/DML-Operations-on-Carbon.md (hunk 3 above)
Comment (lion-x) [inline docs/DML-Operations-on-Carbon.md]: I add a note, ref to the JAVA SimpleDateFormat Class Doc. It provides more details.

**Comment by lion-x** [comment]
CI passed.
http://136.243.101.176:8080/job/ApacheCarbonManualPRBuilder/545/

**Pull request merged**